pyvo/registry/regtap.py

```diff
@@ -384,11 +388,12 @@ class Interface:
         "ivo://ivoa.net/std/sla": sla.SLAService,
         "ivo://ivoa.net/std/tap": tap.TAPService}
 
-    def __init__(self, access_url, standard_id, intf_type, intf_role):
+    def __init__(self, access_url, standard_id, intf_type, intf_role, capability_description):
         self.access_url = access_url
         self.standard_id = standard_id or None
         self.type = intf_type or None
         self.role = intf_role or None
+        self.capability_description = capability_description or None
```

Comment:
I've changes these to default to `None` in #507, might do that for the new parameter here, so we'll have fewer things to refactor in v1.6

Reply:
```suggestion
    def __init__(self, access_url, standard_id, intf_type, intf_role, capability_description=None):
        self.access_url = access_url
        self.standard_id = standard_id or None
        self.type = intf_type or None
        self.role = intf_role or None
        self.capability_description = capability_description
```